src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/MediaPicker/ImageListQuery/ImageListQuery.jsx

```diff
@@ -14,13 +15,20 @@ const ImageListQueryCmp = ({
     selectedPath,
     initialSelection
 }) => {
-    const {data, error, loading} = useQuery(MediaPickerImages, {
+    const {data, error, loading, refetch} = useQuery(MediaPickerImages, {
         variables: {
             typeFilter: ['jmix:image'],
             path: selectedPath
         }
     });
 
+    useEffect(() => {
+        registry.add('refetch-image-list', {
+            type: 'refetch-upload',
+            refetch: refetch
+        });
+    });
```

Comment:
```suggestion
    }, []);
```

Are you sure this code should be executed each render ?

Reply:
yes, it works.

Adding your suggesting break things :3